rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -458,6 +461,11 @@ rmw_create_client(
   cleanup_datawriter.cancel();
   cleanup_datareader.cancel();
   cleanup_info.cancel();
+  if (TRACETOOLS_TRACEPOINT_ENABLED(rmw_client_init)) {
+    rmw_gid_t gid{};
+    rmw_fastrtps_shared_cpp::copy_from_fastrtps_guid_to_byte_array(info->reader_guid_, gid.data);
```

Comment:
using response reader (response topic subscriber) GID as client identification via tracepoint.
this actually matches with `service introspection event identification`, i think that is good to have consistent identification between service introspection event and tracepoint for user.

related to https://github.com/ros2/rmw_fastrtps/pull/781

Reply:
yeah, I actually tried to use the GID used for `rmw_get_gid_for_client()` when possible (which is what was changed in #781), although in the case of `rmw_cyclonedds`, that didn't work.